config/environments/development.js

```diff
@@ -8,6 +8,17 @@ exports.getConfig = function() {
     saltSecret: 'secret token',
     secret: 'secret',
 
-    origin: 'http://localhost:3333'
+    origin: 'http://localhost:3333',
+
+    attachments: {
+      // Make sure that all directories here have a trailing slash
+      urlDir: 'http://frf.creagenics.com:3000/attachments/original/',
```

Comment:
Can you change development settings to localhost please? That'd make it super easy to run server and html-client for newcomers.